placetobe-header.html

```diff
@@ -235,10 +239,20 @@
         menuOpen: {
           type: Boolean,
           value: false
+        },
+
+        campaignArrayHidden: {
+          type: Boolean,
+          value: true,
+          computed: '_checkCampaignArrayLength(apiCampaignData)'
         }
 
       },
 
+      _checkCampaignArrayLength: function(apiCampaignData) {
```

Comment:
`_computeHasCampaigns(splice) {
return (this. apiCampaignData && apiCampaignData.length > 0);
}